### DISCUSSION TITLE
How to search songs from SoundCloud?

### DISCUSSION THREAD

**Discussion by TheShubhendra** [lavalink-devs/Lavalink#560] [Q&A] [answered]
I know the method to play a song from SoundCloud using song link.
But is there any method to query the song using it's name from SoundCloud ?

**Comment by topi314** [comment, marked as answer]
pass `scsearch:<query>`to the [loadtracks](https://github.com/freyacodes/Lavalink/blob/master/IMPLEMENTATION.md#track-loading-api) endpoint
